dll/cpl/sysdm/general.c

```diff
@@ -62,29 +62,29 @@ static VOID InitLogo(HWND hwndDlg)
     BITMAP logoBitmap;
     BITMAP maskBitmap;
     BITMAPINFO bmpi;
-    HDC hDC = GetDC(hwndDlg);
+    HDC hDC;
     HDC hDCLogo = CreateCompatibleDC(NULL);
     HDC hDCMask = CreateCompatibleDC(NULL);
     HBITMAP hMask, hLogo, hAlphaLogo = NULL;
     COLORREF *pBits;
     INT line, column;
 
+    if (hDC == NULL)
```

Comment:
`hDC` is not initialized here.

Reply:
Oups, my mistake when taking previous batch of comments. I fix